### ISSUE TITLE
Expected date for full evaluation code

### ISSUE THREAD

**Issue by TarlanBakirli** [amyxlase/relpose-plus-plus#4]
Hi, congrats on the great work and thank you for sharing code.

I was wondering if there is an estimated timeline for the release of the complete evaluation code. Thank you for any 
information you can provide on this matter.

**Comment by ttsesm** [comment]
We would also be interested on the evaluation protocol that is followed. Because from a quick test that we have applied on the subset of the Co3D (https://github.com/facebookresearch/co3d#single-sequence-dataset-subset) and using the pre-trained models the accuracy for within 15 degrees of tolerance that we get is approx. 21% which is a lot bellow the 70%-80% reported in the paper :thinking:

**Comment by jasonyzhang** [comment]
Evaluation code will be available soon

**Comment by Calimero** [comment]
Thank you so much!

**Comment by TarlanBakirli** [comment]
Thank you!

**Comment by jasonyzhang** [comment]
Evaluation code has been updated. Please follow directions [here](https://github.com/amyxlase/relpose-plus-plus/blob/main/eval.md).
